test/integration/webhook/v1alpha1/workload_test.go

```diff
@@ -70,3 +70,76 @@ var _ = ginkgo.Describe("Workload defaulting webhook", func() {
 		})
 	})
 })
+
+var _ = ginkgo.Describe("Workload validating webhook", func() {
+	ginkgo.Context("When creating a Workload", func() {
+		ginkgo.It("Should validate spec.podSets.count", func() {
+			ginkgo.By("Creating a new Workload")
+			workload := testing.MakeWorkload("workload2", ns.Name).Obj()
```

Comment:
nit: no need to specify the workload name different with each other in `It` node.

Reply:
Okay I would use a constant name later.

Reply:
Please do it now. This is simple enough and you might forget later.